Strain/StrainVisualization.h

```diff
@@ -0,0 +1,75 @@
+#ifndef STRAIN_VISUALIZATION_H
+#define STRAIN_VISUALIZATION_H
+
+#include <SFML/Graphics.hpp>
+
+#include "StrainGauge.h"
+
+
+/**
+ * \brief Class of visualising data from the strain gauge
```

Comment:
**(must be changed, repeatedly)**: add periods at the end of the comments.

Reply:
Done